### ISSUE TITLE
No longer recording datastore telemetry events `updated` and `deleted`

### ISSUE THREAD

**Issue by irrationalagent** [mozilla-lockbox/lockbox-extension#656]
https://sql.telemetry.mozilla.org/queries/49016/source#table

Starting last thu (2018-03-08, same day as 0.1.8 release) we stopped receiving events with `category` == 'lockboxv1', `method` == 'updated' and `object` == 'datastore'. We are still receiving  `added` datastore events.

A local test on my machine confirms it is not recording (running in guest mode):
<img width="1187" alt="screen shot 2018-03-13 at 10 40 40 am" src="https://user-images.githubusercontent.com/5625313/37356184-05730b2a-26ab-11e8-991a-10c2c03c4fa2.png">

1. We are still getting the front-end (`category` == 'manage') events `itemUpdated`, `itemUpdating`, and `itemDeleting` (we never implemented an `itemDeleted`)

2. We are also still getting the datastore event `added`, so it doesn't seem to be a general issue with datastore events. 

after looking at the console log, I'm seeing 

`Invalid extra key for event ["lockboxv1", "itemSelected", "manage"].  (unknown)`
`Invalid extra key for event ["lockboxv1", "deleted", "datastore"].  (unknown)`

so it appears that `itemSelected` is also affected, and that we are trying to set the `extra` field incorrectly. I signed in to make sure that the `fxa_uid` was getting added to `extra` for the events that are firing and it was.

Looking at bootstrap.js, `itemid` was missing from the registration of `itemSelected`; although in `src/webextension/list/popup/telemetry.js` it looks as if the item id **is** passed to the extra field when the event is recorded. I figured this was the problem, but when I added `itemid` to the extras field in bootstrap.js I still got `Invalid extra key for event ["lockboxv1", "itemSelected", "doorhanger"].`

I also tried removing support for `fields` in the extra keys for the datastore events (also by modifying `src/webextension/background/datastore.js` ) but that didn't seem to solve the problem either.

tl;dr I think something is going wrong when adding the itemid to the extra field for some events, but I'm not sure yet.

**Comment by irrationalagent** [comment]
also getting 
`Invalid extra key for event ["lockboxv1", "passwordCopied", "doorhanger"].`
`Invalid extra key for event ["lockboxv1", "usernameCopied", "doorhanger"]`

so those events seem to be affected as well

**Comment by devinreams** [comment]
I'll move this to "to do" since it's important to fix what's now seemingly broken.

Thanks for the detailed report @irrationalagent.

**Comment by jimporter** [comment]
Would a regression range help here? It sounds like the problem is something that recently changed, but it's hard to tell since our tests here are sadly incomplete.
